compiler/rustc_typeck/src/check/upvar.rs

```diff
@@ -924,7 +996,7 @@ fn var_name(tcx: TyCtxt<'_>, var_hir_id: hir::HirId) -> Symbol {
 /// (Note: CaptureInfo contains CaptureKind and an expression that led to capture it in that way)
 ///
 /// If both `CaptureKind`s are considered equivalent, then the CaptureInfo is selected based
-/// on the `CaptureInfo` containing an associated expression id.
+/// on the `CaptureInfo` containing an associated path_expr_id.
```

Comment:
I was thinking this to be `caputre_kind_expr`.

And can you leave a comment saying it's the caller's duty to figure out which `path_expr_id` to use